src/wwwroot/styles/views/_settings.scss

```diff
@@ -397,9 +397,9 @@
   .plan--box-container{
     padding-top: 5em;
     ul {
-      padding: 0; 
-      margin: auto; 
-      display: inline-block;
+      padding: 0;
+      margin: auto;
+      display: block;
```

Comment:
Did you review this change with a CSS expert?

Reply:
I'haven't but I've asked @cbernat to review it

Reply:
👍